go.mod

```
@@ -39,9 +39,9 @@ require (
 	github.com/onsi/ginkgo/v2 v2.6.0
 	github.com/onsi/gomega v1.24.1
 
-	//pinned openshift to release-4.5 branch
-	github.com/openshift/api v0.0.0-20200526144822-34f54f12813a
-	github.com/openshift/client-go v0.0.0-20200521150516-05eb9880269c
+	//pinned openshift to release-4.15 branch
```

This is technically not 'pinned'
```suggestion
	// use openshift from 'release-4.15' branch
```